test/framework/cluster.go

```diff
@@ -2177,18 +2174,6 @@ func dumpFile(description, path string, t T) {
 	t.Logf("%s:\n%s\n", description, string(b))
 }
 
-func (e *ClusterE2ETest) setFeatureFlagForUnreleasedKubernetesVersion(version v1alpha1.KubernetesVersion) {
-	// Update this variable to equal the feature flagged k8s version when applicable.
-	// For example, if k8s 1.26 is under a feature flag, we would set this to v1alpha1.Kube126
-	unreleasedK8sVersion := v1alpha1.Kube129
```

Comment:
I think we usually just set this to an empty version instead of deleting the entire method

Reply:
I did whatever [this PR](https://github.com/aws/eks-anywhere/pull/6830/files#diff-4438221d080eeaeb7e58673f2b24e69bf482eb366de82660ae94342614f9ef7dL2172) was doing for 1.28, but I actually like this better. Fewer additions and deletions.